formats/email_alert_signup/publisher/links.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "related"
+  ],
+  "properties": {
```

Comment:
Looking at the [email-alert-frontend pull request](https://github.com/alphagov/email-alert-frontend/pull/1), it doesn't look like any of these properties are actually needed to generate an email alert. If so, is there another reason to include `related`, `organisations` and `people`? If there isn't, we shouldn't include them in the schema.